packages/sdk/src/pubkey.ts

```diff
@@ -1,11 +1,79 @@
 import { Secp256k1 } from "@iov/crypto";
-import { Encoding } from "@iov/encoding";
+import { Bech32, Encoding } from "@iov/encoding";
+import equal from "fast-deep-equal";
 
-import { PubKey, pubkeyType } from "./types";
+import { Bech32PubKey, PubKey, pubkeyType } from "./types";
 
 export function encodeSecp256k1Pubkey(pubkey: Uint8Array): PubKey {
   return {
     type: pubkeyType.secp256k1,
     value: Encoding.toBase64(Secp256k1.compressPubkey(pubkey)),
   };
 }
+
+export type CosmosPubkeyBech32Prefix = "cosmospub" | "cosmosvalconspub" | "cosmosvaloperpub";
+const validPubkeyPrefixes = ["cosmospub", "cosmosvalconspub", "cosmosvaloperpub"];
```

Comment:
Didn't even see this file. Definitely better place for the logic

Reply:
I only created it yesterday when extracting `encodeSecp256k1Pubkey`